lib/carin_for_blue_button_test_kit/generator/templates/validation.rb.erb

```diff
@@ -12,6 +12,17 @@ module CarinForBlueButtonTestKit
       description %(<%=
         description %>
       )
+      <%- if test_id == 'c4bb_v200_eob_validation_test' -%>
```

Comment:
I don't think we can include 131, 132 - these seem more like business rules that we can't confirm through capability statement validation

Reply:
Sounds good! I will remove 131, just to confirm, I don't think I had 132 in there as it is an attestation already?

Reply:
yep - misread by me. 132 not present.